init.lua

```diff
@@ -170,10 +170,12 @@ require('lazy').setup({
     'lukas-reineke/indent-blankline.nvim',
     -- Enable `lukas-reineke/indent-blankline.nvim`
     -- See `:help indent_blankline.txt`
-    opts = {
-      char = '┊',
-      show_trailing_blankline_indent = false,
-    },
+    config = function()
+      require('ibl').setup {
+        char = '┊',
+        show_trailing_blankline_indent = false,
+      }
+    end,
```

Comment:
This can be done easier by setting `main = 'ibl'` in the plugin config. Also it seems `show_trailing_blankline_indent = false` is the default now.

Reply:
The `main = "ilb"` solution (it's in the v3 migration docs) seems to be working for me, the conig function solution didn't work for me. 🤷‍♂️ 

![image](https://github.com/nvim-lua/kickstart.nvim/assets/6851336/2d16594a-cf40-495a-b6c6-dbb3ff3be0dc)

Reply:
also the `main = 'ibl'` is the way how [upstream](https://github.com/lukas-reineke/indent-blankline.nvim/wiki/Migrate-to-version-3) recommends it